apps/protailwind/src/pages/tutorials/[module]/[section]/[lesson]/index.tsx

```diff
@@ -36,32 +40,36 @@ export const getStaticProps: GetStaticProps = async (context) => {
 export const getStaticPaths: GetStaticPaths = async (context) => {
   const tutorials = await getAllTutorials()
 
-  const paths = tutorials.reduce((acc: any[], tutorial: any) => {
-    return [
-      ...acc,
-      ...tutorial.lessons.map((exercise: any) => {
-        return {
-          params: {
-            module: tutorial.slug.current,
-            lesson: exercise.slug,
-          },
-        }
-      }),
-    ]
-  }, [])
+  // flatMap to extract lessons in sections from tutorials
+  const paths = tutorials.flatMap((tutorial: any) => {
+    return (
+      tutorial.sections?.flatMap((section: any) => {
+        return (
+          section.lessons?.map((lesson: any) => ({
+            params: {
+              module: tutorial.slug.current,
+              section: section.slug,
+              lesson: lesson.slug,
+            },
+          })) || []
+        )
+      }) || []
+    )
+  })
```

Comment:
This is a clean solution for the data. I was over complicating the template on my end